docs/website/docs/developers/performance/profiling-with-tracy.md

```diff
@@ -394,9 +394,23 @@ sudo sh -c "ulimit -n <bigNum> && <myTracyInstrumentedProgram>"
 
 ### Building Tracy from source
 
-#### Install dependencies
+First, refer to the upstream build instructions at either the
+<https://github.com/wolfpld/tracy/> repository itself or the
+[Tracy PDF manual](#the-tracy-manual).
 
-##### Do you need capstone-next?
+For example, to build the profiler GUI from an IREE checkout:
+
+```bash
+# Build using CMake:
+cd third_party/tracy
+cmake -B profiler/build -S profiler -DCMAKE_BUILD_TYPE=Release
+cmake --build profiler/build --parallel --config Release
+
+# Now launch the profiler:
+./profiler/build/tracy-profiler
+```
+
+#### Additional Capstone dependencies for CPU code disassembly
```

Comment:
@bjacob I'm keeping these capstone docs for now. Sounds like we could remove them after updating Tracy to use a newer version?